packages/commercetools/api-client/src/index.ts

```diff
@@ -19,6 +19,7 @@ import customerSignMeUp from './api/customerSignMeUp';
 import customerSignMeIn from './api/customerSignMeIn';
 import customerSignOut from './api/customerSignOut';
 import getStorage from './helpers/createCommerceToolsLink/getStorage';
+import getMyOrders from './api/getMyOrders';
```

Comment:
this is clearer for index :)
```suggestion
export * from './api/getMyOrders';
```

Reply:
I agree, just in this particular case in ends with some unexpected error thus I'd like to handle this change for entire file separately. :)